### ISSUE TITLE
firefox css not loading in 0.31.0v 

### ISSUE THREAD

**Issue by Zyzto** [BookStackApp/BookStack#2443]
**Describe the bug**
site doesn't load in correctly on firefox 84.0.1

**Expected behavior**
to load the site in correct style

**Screenshots**
![image](https://user-images.githubusercontent.com/13531154/103538419-3889ec00-4ea7-11eb-9d8b-93ba30b39a19.png)
![image](https://user-images.githubusercontent.com/13531154/103538619-974f6580-4ea7-11eb-9ad3-30f20a04cfa6.png)

**Your Configuration (please complete the following information):**
 - Exact BookStack Version (Found in settings): 0.31.0
 - PHP Version: 7.2.24 and tried php-remi 7.3.5
 - Hosting Method (Nginx/Apache/Docker): Apache

**Comment by ssddanbrown** [comment]
Hi @Zyzto,
I would be ideal to know what version you upgrade from if possible.

Is the `APP_URL` configured in your `.env` file, including the starting `http://` or `https://` part? That would need to match the URL you're accessing BookStack on.

**Comment by Zyzto** [comment]
@ssddanbrown from 0.30.4

**Comment by ssddanbrown** [comment]
@Zyzto Okay, from your bookstack install can you check that a `public/dist/styles.css` file exists?

Also, in the "Style sheet could not be loaded" warning in shown in your sceenshot, does the URL match (`<APP_URL>/dist/styles.css`)

**Comment by Zyzto** [comment]
@ssddanbrown after 2h of missing with firefox yeah, I hate it.
it doesn't work with dns that well, it has the ip but unable to access the site.
what I entered as the name is **Wiki** and it works but when I use the FQDN **wiki.domain.tld** even tho it has the IP in config page inside firefox it's just can't connect. and I tested on multiple devices and multiple OSs but still the same.

thanks for tying to help
I will close this issue as it has no relation to bookstack and more of firefox issues.
